src/test/kotlin/org/jmailen/gradle/kotlinter/functional/KotlinProjectTest.kt

```diff
@@ -140,18 +123,6 @@ internal class KotlinProjectTest : WithGradleTest.Kotlin() {
             repositories {
                 jcenter()
             }
-
-            import org.jmailen.gradle.kotlinter.tasks.LintTask
-
-            task ktLint(type: LintTask) {
-                source files('src')
-                reports = ['plain': file('build/lint-report.txt')]
-                indentSize = 5
-                continuationIndentSize = 7
-                experimentalRules = true
-                disabledRules = ["final-newline"]
-                editorConfigPath = project.rootProject.file(".editorconfig")
```

Comment:
Here are 2 things I'd like to point out:
1. From now on, all tasks treat `.editorconfig` file as one their inputs so the file has to exist. This test started to fail because of that. I decided to move tests related with "Custom Tasks" to separate class and started creating `.editorconfig` file in each test.
2. `editorConfigPath` name suggested it expects a `String` value but in fact, we were passing a File instance.
I changed the task input to use FileProperty, but leaved the name for compatibility reasons.